src/wh_client_crypto.c

```diff
@@ -2072,4 +2072,492 @@ int wh_Client_CmacCancelableResponse(whClientContext* c, Cmac* cmac,
     return ret;
 }
 #endif /* WOLFSSL_CMAC */
+
+#ifdef HAVE_DILITHIUM
```

Comment:
Yuck.  This should be spelled HAVE_MLDSA or HAVE_FIPS204.  No change.  Just complaining